cmd/docker-generate/generate.go

```diff
@@ -1,12 +1,13 @@
 package main
 
 import (
-	"github.com/urfave/cli/v2"
+	"github.com/spf13/cobra"
 )
 
-var generateCommand = &cli.Command{
-	Name: "generate",
-	Subcommands: []*cli.Command{
-		manifestCommand,
-	},
+func generateCommand() *cobra.Command {
```

Comment:
This func does not have a unit test yet. (Under [Github Actions](https://github.com/notaryproject/notation/actions/runs/2677789762), there's the notation-coverage-report showing coverage percentage of the current build.)